test/parallel/test-child-process-pipe-dataflow.js

```diff
@@ -19,14 +20,15 @@ const MB = KB * KB;
 {
   tmpdir.refresh();
   const file = path.resolve(tmpdir.path, 'data.txt');
-  const buf = Buffer.alloc(MB).fill('x');
+  let buf = Buffer.alloc(MB).fill('x');
 
   // Most OS commands that deal with data, attach special
   // meanings to new line - for example, line buffering.
   // So cut the buffer into lines at some points, forcing
   // data flow to be split in the stream.
+  const eol_len = os.EOL.length;
   for (let i = 0; i < KB; i++)
-    buf[i * KB] = 10;
+    buf = buf.fill(os.EOL, i * KB, (i * KB + eol_len));
```

Comment:
I think just replacing `buf[i * KB] = 10` with `buf.write(os.EOL, i * KB)` should be fine, too, and is a little more direct.

Reply:
@addaleax, thanks. Replaced `but.fill(os.EOL, i * KB, (i * KB + eol_len))` with your suggestion.